workers/unity/Assets/Gdk/Core/Error/ForwardingDispatcher.cs

```diff
@@ -0,0 +1,67 @@
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class ForwardingDispatcher : ILogDispatcher
+    {
+        private Connection connection;
+
+#if UNITY_EDITOR
+        private readonly LoggingDispatcher loggingDispatcher = new LoggingDispatcher();
+#endif
+
+        public void SetConnection(Connection connection)
+        {
+            this.connection = connection;
+        }
+
+        public void HandleLog(LogType type, LogEvent logEvent)
+        {
+#if UNITY_EDITOR
+            if (Application.isEditor)
+            {
+                loggingDispatcher.HandleLog(type, logEvent);
+            }
+#endif
+
+            if (connection == null)
+            {
+                return;
+            }
+
+            LogLevel logLevel;
+            var message = logEvent.ToString();
+
+            switch (type)
+            {
+                case LogType.Error:
+                    logLevel = LogLevel.Error;
+                    break;
+                case LogType.Warning:
+                    logLevel = LogLevel.Warn;
+                    break;
+                case LogType.Log:
+                    logLevel = LogLevel.Info;
```

Comment:
Do we want to log everything by default? I'd argue that warnings and above only by default. Sending all your debug information to the runtime is mostly just noise. Especially as right now we are mostly running things in the editor.